pkg/certificate/manager.go

```diff
@@ -15,10 +17,13 @@ import (
 	"github.com/openservicemesh/osm/pkg/messaging"
 )
 
+var group singleflight.Group
```

Comment:
i'd consider scoping this to the certmanager, in case there are ever multiple

Reply:
Good idea. I think I might leave this as-is for this PR unless this is a blocking comment.

Reply:
I would say it’s a bug not to, since if 2 Cert manager objects are created the cache would not be populated correctly. 

It’s a 1-2 line fix, so I would push for this in this pr

Reply:
Updated